packages/datasets/src/loadPageDataset.ts

```diff
@@ -0,0 +1,38 @@
+import { MongoDbPageStore, PersistedPage } from "mongodb-rag-core";
+
+type PageDatasetEntry = Pick<
+  PersistedPage,
+  "url" | "body" | "metadata" | "title" | "sourceName" | "updated" | "format"
+>;
+
+/**
+  @param pageStore - datastore for pages
+  @param dataSourceRegex - regular expression to filter pages by dataSource
+  @param forbiddenUrls - set of urls to exclude from the dataset
+ */
+export async function loadPagesDataset(
+  pageStore: MongoDbPageStore,
+  dataSourceRegex: RegExp,
+  forbiddenUrls: string[]
+): Promise<PageDatasetEntry[]> {
```

Comment:
Could we switch this to use an args object w/ corresponding type instead of positional args?